config/app.default.php

```diff
@@ -468,6 +468,20 @@
         ],
     ],
 
+    /**
+     * Modules accesses per role(s)
+     */
+    // 'Roles' => [
```

Comment:
Can we use `AccessControl` as key instead of `Roles`? Or `Access` or `ACL`... 

[superminor] I would not use `admin` in these examples because it's not yet available and will probably be `admin`-role only by default